packages/firebase_auth/firebase_auth/lib/src/firebase_auth.dart

```diff
@@ -15,6 +15,10 @@ class FirebaseAuth extends FirebasePluginPlatform {
   // instance with the default app before a user specifies an app.
   FirebaseAuthPlatform? _delegatePackingProperty;
 
+  /// To set "persistence" on web, it is now required on web v9 JS SDK to pass the value on the initializeAuth() call.
```

Comment:
```suggestion
  // To set "persistence" on web, it is now required on the v9.0.0 or above Firebase JS SDK to pass the value on calling `initializeAuth()`.
```

`//` double slash for internal documentation